Specs/Core/OrientedBoundingBoxSpec.js

```diff
@@ -101,6 +101,8 @@ defineSuite([
         var result = rotatePositions(positions, Cartesian3.UNIT_Z, CesiumMath.PI_OVER_FOUR);
         var points = result.points;
         var rotation = result.rotation;
+        rotation[1] = -rotation[1];
```

Comment:
I think the result.rotation matrix computed by the quaternion was rotating the opposite direction.  Negating these values makes it equal to box.halfAxes.  Since the positions are symmetrical, the box is the same for both.  @bagnell is this an okay solution?